test/project/git/GitProject.test.ts

```diff
@@ -152,7 +152,7 @@ ding dong ding
         const gs2 = await p2.gitStatus();
         assert(gs1.sha === gs2.sha);
         await p.release();
-    }).timeout(5000);
+    }).timeout(10000);
```

Comment:
I've had occasional test failures due to timeouts here, can revert if you'd like to keep the timeout as it was.